ompi/mca/coll/monitoring/configure.m4

```diff
@@ -16,7 +16,7 @@
 AC_DEFUN([MCA_ompi_coll_monitoring_CONFIG],[
     AC_CONFIG_FILES([ompi/mca/coll/monitoring/Makefile])
 
-    AS_IF([test MCA_BUILD_ompi_common_monitoring_DSO_TRUE == ''],
+    AS_IF([test "$MCA_BUILD_ompi_common_monitoring_DSO_TRUE" == ''],
```

Comment:
Wait -- isn't using `==` with `test` non-portable?  I.e., doesn't this get us in trouble with the BSD flavors of `test`?

Reply:
Good question, I just added quoting and '$' to expand the variable.  It does appear that other instances of `test` with  `AS_IF` and  `MCA_BUILD_`   are using single `=`.  I'll defer to experts.

Reply:
According to the rest of OMPI code Jeff is right we should use a single = instead.